### PR TITLE
Always display "Related product" title

### PR DESCRIPTION
**Pull request by idnovate**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description | Display a title even if there's not any related product
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no

Before
image

After
image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19239)

### CODE REVIEW

**Comment by idnovate** [comment]
Before
image

After
image

File: src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_related_products.html.twig [hunk 1]
```diff
@@ -22,10 +22,12 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-<div id="related-content" class="row {{ form.vars.value.data|length == 0 ? 'hide':'' }}">
+<div class="row">
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_related_products.html.twig]:
```suggestion
<div  id="related-title" class="row">
```

That permits to identify the title ;)

**Comment by Progi1984** [comment]
@TristanLDD Do you valid this change ?

**Review by PierreRambaud — DISMISSED** [review]

**Comment by prestascott** [comment]
Hi @Progi1984,

By adding "Related product" as a title it will be easier to identify related products that have been added. 
I valid this change :)

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @idnovate,

It is ok :heavy_check_mark: 
But we need a confirmation from the Product team.
Ping @PrestaShop/prestashop-product-team what do you think?
Before
image
After
image

Thanks!

**Comment by MatShir** [comment]
@idnovate Nice improvement, we wanted to make this change when we migrated the page but it seems you were faster. Thanks for your contribution

**Comment by Progi1984** [comment]
Ping @khouloudbelguith

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @idnovate

**Pull request merged**